test/webrtc_test.exs

```diff
@@ -132,4 +132,102 @@ defmodule ExSDP.WebRTCTest do
              version: 0
            }
   end
+
+  test "SDP with rtx and FID ssrc-group" do
```

Comment:
I would externd the previous test

Reply:
I didn't want to do that, because the SDP is an unmodified version of what is sent to Jitsi Meet by the browser and wanted to keep it that way. Adding new options to the SDP above would require adjusting some values while replacing the SDP with a new one would force me to regenerate the whole output.

I do believe this to be a separate test case

Reply:
Does it mean we are going to add a separate test for each newly added attribute? 

You can also generate new expected structure in two/three lines of code

```elixir
File.read!(chrome_jitsi_offer.txt)
|> ExSDP.parse()
```